src/test/java/spoon/test/ctType/CtTypeParameterTest.java

```diff
@@ -86,7 +86,7 @@ private void checkTypeParamErasureOfExecutable(CtTypeParameter typeParam) throws
 		}
 		Class<?> paramType = declExec.getParameterTypes()[paramIdx];
 		// contract the type erasure given with Java reflection is the same as the one computed by spoon
-		assertEquals("TypeErasure of executable param " + getTypeParamIdentification(typeParam), paramType.getName(), typeParam.getTypeErasure().toString());
+		assertEquals("TypeErasure of executable param " + getTypeParamIdentification(typeParam), paramType.getTypeName(), param.getType().getTypeErasure().toString());

```

Comment:
it's strange to change both the expected value and the tested one.

what is the new value of `typeParam.getTypeErasure()` after the fix?